modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/jest.config.js

```diff
@@ -9,11 +9,8 @@
  * distribution rights of the Software.
  */
 
-const enzyme = require('enzyme');
-const EnzymeAdapter = require('enzyme-adapter-react-16');
-
 window.AUI = () => ({
-	use: (module, callback) => callback(),
+	use: (module, callback) => callback(module),
 });
 
 const languageMap = {
```

Comment:
@wincent, out of the scope of this, but can you check the `window.Liferay` mock? I doubt all of that is necessary and it's wiping out our default one... maybe there's something worth moving to our [Liferay mock}(https://github.com/liferay/liferay-npm-tools/blob/master/packages/liferay-npm-scripts/src/jest/mocks/Liferay.js)

Reply:
@albinojunior, could you check this as well? You might not need most of the things in `window.Liferay = {...}`

Reply:
Ha, I didn't even look at this code. You're right that most of this isn't needed:

```
window.Liferay = {
	Language: {
		get: (key) => {
			if (languageMap[key]) {
				return languageMap[key];
			}

			return key;
		},
	},
	Session: {
		extend: () => {},
	},
	ThemeDisplay: {
		getBCP47LanguageId: () => 'en-US',
		getLanguageId: () => 'en_US',
		getPathThemeImages: () => 'http://localhost:8080/o/admin-theme/images',
		getUserId: () => '123',
		getUserName: () => 'Test Test',
	},
	authToken: 'auth',
};
```

Of the above, [the mock](https://github.com/liferay/liferay-npm-tools/blob/e5fffe85f7e428da43903a2544a1c97d2e8abfe9/packages/liferay-npm-scripts/src/jest/mocks/Liferay.js) currently defines all but:

- `Liferay.Session.extend()`
- `Liferay.ThemeDisplay.getUserId()`
- `Liferay.ThemeDisplay.getUserName()`

Our approach when we made the mock was to make a minimal stub for everything that was currently needed in the test suite at the time, rather than throwing everything but the kitchen sink in there; but when there is actual usage, we can extend it, so the above would be reasonable candidates for inclusion.

Made a ticket for that here: https://github.com/liferay/liferay-npm-tools/issues/502

Reply:
Hey @wincent and @jbalsas, I may clean up these unnecessary mocks when I update the tests to removing the warnings and after that is included [liferay/liferay-npm-tools#502](https://github.com/liferay/liferay-npm-tools/issues/502).

Reply:
FWIW @albinojunior, because that issue includes a backlink to this PR, what I usually do for changes like that is make the change in liferay-npm-scripts _and_ make the corresponding change in liferay-portal so that I can prove to myself that it actually works, so you probably don't have to worry about touching the mock.